src/handler/structures/context.ts

```diff
@@ -140,7 +140,7 @@ export default class Context {
                     .then(() => i.fetchReply());
             }),
             this.oMsg.map(m => {
-                return m.reply(content as string | ReplyMessageOptions);
+                return m.reply(content as string | MessageReplyOptions);
```

Comment:
this fixes the 14.4 break for autocomplete right?

Reply:
yes